packages/nimble-components/src/label-provider/base/tests/label-providers.mdx

```diff
@@ -29,3 +30,9 @@ Provides labels for all Nimble components besides the table component.
 Provides labels for the `nimble-table` and all associated table sub-components / column types.
 
 <Canvas of={tableLabelProvider} sourceState="none" />
+
```

Comment:
Let's put the Rich Text Editor before the Table (alphabetical order). That will match the ordering in the tree view, too.

Reply:
Updated